dist/ResizeObserver.js

```diff
@@ -738,7 +738,7 @@
        * @param target Element to be observed.
        */
       ResizeObserverSPI.prototype.observe = function (target) {
-          if (!arguments.length) {
+          if (target === undefined) {
```

Comment:
+1 to that, `arguments` is p much always a mistake, and has so many fiddly rules